### PR TITLE
Creating ABI RSR files for GOES 18 & 19 (FM3 & 4)

### LINKED ISSUE #132: Add GOES-18 and GOES-19 RSRs
#### Code Sample, a minimal, complete, and verifiable piece of code

No code sample, sorry.

#### Problem description

Trying to process GOES-18 simulated data with Satpy produces an exception in pyspectral when it tries to read the RSR data. The SRFs are available at:

https://ncc.nesdis.noaa.gov/GOESR/ABI.php

FM3 is G18 and I guess FM4 is G19.

#### Expected Output

Success!

#### Actual Result, Traceback if applicable

```
  File "/home/shared/bin/geo2grid_v_1_0_2/lib/python3.7/site-packages/satpy/scene.py", line 843, in _generate_composite
    **self.attrs)
  File "/home/shared/bin/geo2grid_v_1_0_2/lib/python3.7/site-packages/satpy/composites/__init__.py", line 575, in __call__
    red.data)
  File "/home/shared/bin/geo2grid_v_1_0_2/lib/python3.7/site-packages/pyspectral/rayleigh.py", line 248, in get_reflectance
    wvl = self.get_effective_wavelength(bandname)
  File "/home/shared/bin/geo2grid_v_1_0_2/lib/python3.7/site-packages/pyspectral/rayleigh.py", line 187, in get_effective_wavelength
    rsr = RelativeSpectralResponse(self.platform_name, self.sensor)
  File "/home/shared/bin/geo2grid_v_1_0_2/lib/python3.7/site-packages/pyspectral/rsr_reader.py", line 104, in __init__
    "platform name and sensor or filename must be specified")
```

#### Versions of Python, package at hand and relevant dependencies

Thank you for reporting an issue !

### PR DESCRIPTION
**Pull request by adybbroe**
Add support for GOES-18 and 19 RSR data.

 - [x] Closes #132 
 - [ ] Tests added 
 - [x] Tests passed: Passes ``pytest pyspectral`` 
 - [x] Passes ``flake8 pyspectral`` 
 - [x] Fully documented

### CODE REVIEW

**Comment by adybbroe** [comment]
Below you see the new RSR for GOES-18 and -19 compared to GOES-16:

rsr_band_1200_1400_goes_16_19
rsr_band_1200_1400_goes_16_18
rsr_band_1000_1250_goes_16_19
rsr_band_1000_1250_goes_16_18
rsr_band_0600_1050_goes_16_19
rsr_band_0600_1050_goes_16_18

File: pyspectral/utils.py [hunk 1]
```diff
@@ -222,10 +70,10 @@
                'MTG-I1': 'fci'
                }
 
-HTTP_PYSPECTRAL_RSR = "https://zenodo.org/record/4305549/files/pyspectral_rsr_data.tgz"
+HTTP_PYSPECTRAL_RSR = "https://zenodo.org/record/5797957/files/pyspectral_rsr_data.tgz"
```
Comment (gerritholl) [inline pyspectral/utils.py:73]: `HTTPS_PYSPECTRAL_RSR` perhaps?

File: rsr_convert_scripts/abi_rsr.py [hunk 2]
```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 #
-# Copyright (c) 2016 - 2018 Pytroll developers
+# Copyright (c) 2016 - 2021 Pytroll developers
 #
 # Author(s):
 #
-#   Adam.Dybbroe <adam.dybbroe@smhi.se>
+#   Adam Dybbroe <Firstname.Lastname@smhi.se>
```
Comment (gerritholl) [inline rsr_convert_scripts/abi_rsr.py:8]: To prevent spam?

File: rsr_convert_scripts/abi_rsr.py [hunk 3]
```diff
@@ -69,27 +67,21 @@ def __init__(self, bandname, platform_name):
...
+        resp = data[2]
 
         self.rsr = {'wavelength': wvl, 'response': resp}
 
 
 def main():
-    """Main"""
-    for platform_name in ['GOES-16', 'GOES-17', ]:
+    """Create pyspectral hdf5 RSR files for all GOES sats."""
+    for platform_name in ['GOES-18', 'GOES-19', ]:
```
Comment (gerritholl) [inline rsr_convert_scripts/abi_rsr.py:84]: Why not all of GOES-16, -17, -18, -19 (also for historical reference)?

**Review by mraspaud — APPROVED** [review]
No more comments than what @gerritholl had, LGTM

File: rsr_convert_scripts/abi_rsr.py (hunk 2 above)
Comment (adybbroe) [reply rsr_convert_scripts/abi_rsr.py:8]: Yes, that was the intention. We should eventually add a AUTHORS.md to Pyspectral so we could remove these though.

File: pyspectral/utils.py (hunk 1 above)
Comment (adybbroe) [reply pyspectral/utils.py:73]: Ah, yes, well. I will leave this for now

File: rsr_convert_scripts/abi_rsr.py (hunk 3 above)
Comment (adybbroe) [reply rsr_convert_scripts/abi_rsr.py]: Good point, done! Thx

**Pull request merged**